streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -48,6 +53,10 @@
 	
 	private final GeneralConfiguration generalConfiguration;
 	
+	private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+	
+	private static final String DB_URL = "jdbc:mysql://localhost:3306/openmrs?autoReconnect=true&useSSL=false";
```

Comment:
This should be configurable based on parameters used for Debezium config.

Reply:
@gitcliff  did you see @bashir2 's comment ??